test/recipes/controls/aws_parallelcluster_config/head_node_base_spec.rb

```diff
@@ -0,0 +1,39 @@
+# Copyright:: 2023 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License").
+# You may not use this file except in compliance with the License. A copy of the License is located at
+#
+# http://aws.amazon.com/apache2.0/
+#
+# or in the "LICENSE.txt" file accompanying this file.
+# This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, express or implied.
+# See the License for the specific language governing permissions and limitations under the License.
+
+control 'head_node_base_configured' do
+  title 'Check the base headnode configuration'
+
+  describe command("sudo su -c 'ls /home/*/.ssh/authorized_keys_cluster'") do
```

Comment:
Nitpick: since it is not immediate to understand what this instruction verifies, I would add a comment.